src/test/java/hudson/plugins/git/GitSCMTest.java

```diff
@@ -157,6 +157,46 @@ private StandardCredentials getInvalidCredential() {
         return new UsernamePasswordCredentialsImpl(scope, id, "desc: " + id, username, password);
     }
 
+    @Test
+    public void testHideGitTagAction() throws Exception {
+        FreeStyleProject project = setupSimpleProject("master");
+        List<UserRemoteConfig> remoteConfigs = GitSCM.createRepoList("https://github.com/jenkinsci/git-plugin", "github");
+        project.setScm(new GitSCM(remoteConfigs,
+                Collections.singletonList(new BranchSpec("master")), false, null, null, null, null));
+
+        GitSCM scm = (GitSCM) project.getScm();
+        final DescriptorImpl descriptor = (DescriptorImpl) scm.getDescriptor();
+        assertFalse("Wrong initial value for hide tag action", scm.isHideGitTagAction());
+        descriptor.setHideGitTagAction(true);
```

Comment:
@MarkEWaite Why are we testing the assertion on the set boolean flag two times?

Reply:
Default value is tested by the first assertion, result of an explicit set(true) is tested by the second assertion.